hyp3_testing/fetch.py

```diff
@@ -0,0 +1,94 @@
+"""Utilities for fetching things from external endpoints
+
+Vendored from hyp3-lib: https://github.com/ASFHyP3/hyp3-lib/blob/develop/src/hyp3lib/fetch.py
```

Comment:
Is it not feasible to make hyp3lib a dependency of hyp3-testing?

Reply:
I was able to build the environment just fine (on an Arm machine) after adding `hyp3lib` above `hyp3_sdk` in `environment.yml`, so I would suggest importing this module from `hyp3lib` instead.